src/server.h

```diff
@@ -253,10 +247,8 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 
 	void deleteParticleSpawner(const std::string &playername, u32 id);
 
-	// Creates or resets inventory
-	Inventory *createDetachedInventory(const std::string &name,
-			const std::string &player = "");
-	bool removeDetachedInventory(const std::string &name);
+	ServerInventoryManager *getInventoryMgr() { return m_inventory_mgr.get(); }
```

Comment:
`ServerInventoryManager *getInventoryMgr() const { ... }` ?

Reply:
not sure we can as sometimes we create detached inventories, will verify that